src/test/java/no/unit/nva/publication/service/ResourcePersistenceServiceTest.java

```diff
@@ -0,0 +1,34 @@
+package no.unit.nva.publication.service;
+
+import org.junit.Test;
+
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.Invocation;
+import javax.ws.rs.client.WebTarget;
+import java.util.UUID;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class ResourcePersistenceServiceTest {
+
+    @Test
+    public void test() {
+
```

Comment:
Does this test anything?

Reply:
Not really, but do you have a better idea to get coverage with Jersey? I will try HttpClient instead.